mesheryctl/internal/cli/root/system/start.go

```diff
@@ -372,14 +372,14 @@ func start() error {
 		}
 
 		if err != nil {
-			return err
-		}
-
-		currCtx.Endpoint = utils.EndpointProtocol + "://" + endpoint.External.Address + ":" + strconv.Itoa(int(endpoint.External.Port))
+			log.Info("service endpoint not discovered. Meshery deployed on default endpoint")
```

Comment:
@metonymic-smokey, Why should we display this message when we do not display the endpoint in the else case? I am not sure how this information is useful to the user on system start?

Reply:
Agreed @hexxdump. I've removed that message entirely.